spec/features/recipes/show_recipe_page_spec.rb

```diff
@@ -0,0 +1,18 @@
+feature "user visits show recipe" do
+  before do
+    FactoryBot.create(:recipe, :scrambled_eggs)
+    visit("/")
+  end
+
+  it "shows user clicked recipe" do
+    page.all('a')[0].click
+    have_css("img")
+    expect(page).to have_content "scrambled eggs"
+    expect(page).to have_content "egg 180 gram"
+    expect(page).to have_content "butter 5 gram"
+    expect(page).to have_content "salt 1 pinch"
+    expect(page).to have_content "pepper 1 pinch"
+    click_on('Back')
+    expect(page).to have_current_path("/recipes")
```

Comment:
We could prepare similar assertion right before our content assertion for `show` page.

Reply:
I just don't know how to specify id in current_path because it's always different (/recipes/584 etc) so I can't find reasonable answer to that.

Reply:
you can do it using regular expression (pattern matching)

```ruby
expect(page).to have_current_path(%r{recipes/\d+})
```

FYI There is cool site that everyone uses for playing with regexps -> http://rubular.com/

OR you can extract the factory creation to `let` statment with bang `!`:

```
let!(:tikka_massala) { FactoryBot.create(:recipe, :tikka_masala) }

before do
  FactoryBot.create(:recipe, :scrambled_eggs)
    visit('/')
  end
end

it do
  expect(page).to have_current_path("/recipes/#{tikka_massala.id}")
end
```